src/typechecker/FStar.TypeChecker.Cfg.fs

```diff
@@ -1188,7 +1190,16 @@ let config' psteps s e =
              ; wpe = Env.debug e (Options.Other "WPE")
              ; norm_delayed = Env.debug e (Options.Other "NormDelayed")
              ; print_normalized = Env.debug e (Options.Other "print_normalized_terms")
-             ; debug_nbe = Env.debug e (Options.Other "NBE")}
+             ; debug_nbe = Env.debug e (Options.Other "NBE")
+             ; erase_erasable_args =
+               (let b = Env.debug e (Options.Other "UNSOUND_EraseErasableArgs") in
+                if b
+                then Errors.log_issue
+                        (Env.get_range e)
+                        (Errors.Warning_WarnOnUse,
+                         "The 'UNSOUND_EraseErasableArgs' setting is for debugging only; it is not sound");
+                b)
+             }
```

Comment:
You can use  `--debug_level UNSOUND_EraseErasableArgs` to enable erasable argument erasure in any call to the normalizer or nbe. This is useful for debugging, particularly in conjunction with the postprocess_with tactic---this will let you see what gets erased at the source level and even within emacs.

But, it is clearly not sound. So, warn in case the option is set.